themes/bootstrap3/templates/record/checkbox.phtml

```diff
@@ -1,5 +1,6 @@
 <label class="record-checkbox hidden-print">
-  <input class="checkbox-select-item" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($this->id) ?>"<?php if (isset($this->formAttr)): ?> form="<?=$this->formAttr ?>"<?php endif; ?> aria-label="<?=$this->transEscAttr('select_item')?>">
+  <?php $title = $this->titleHtml ?? $this->escapeHtmlAttr($driver?->getTitle() ?? $this->id); ?>
+  <input class="checkbox-select-item" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($this->id) ?>"<?php if (isset($this->formAttr)): ?> form="<?=$this->formAttr ?>"<?php endif; ?> aria-label="<?=$title ?>">
```

Comment:
Is it okay for `aria-label` to include HTML? I think by definition, it's possible that `$this->titleHtml` will contain markup (e.g. highlighting-related `<span>`s). While I think it's unlikely that would happen in this particular context, maybe it's worth running a `strip_tags` on the output just to be on the safe side.

Reply:
Done